src/Plugin/Field/FieldType/OgMembershipReferenceItem.php

```diff
@@ -31,6 +31,33 @@ class OgMembershipReferenceItem extends EntityReferenceItem {
   /**
    * {@inheritdoc}
    */
+  public static function defaultFieldSettings() {
+    $settings = parent::defaultFieldSettings();
+    $settings['access_override'] = FALSE;
```

Comment:
@amitaibu Also ported the `access_override` setting. It used to live on the behaviour handler. I think it makes sense to move to the field settings? For things like access hooks, the $field_definition is much more accessible than any widget settings. The widget should almost be irrelevant here in an ideal world?